discourse/benchmarks/bench.rb

```diff
@@ -173,6 +173,10 @@ def bench(path)
   percentiles
 end
 
+def generate_digest
+  Digest::SHA2.hexdigest("#{ENV['POSTGRES_ENV_PG_VERSION']}#{ENV['REDIS_ENV_REDIS_VERSION']}")
```

Comment:
TODO: Add Discourse commit hash.

http://community.rubybench.org/t/discourse-keeps-changing-can-we-freeze-the-version-of-discourse-bench/42